### DISCUSSION TITLE
What messages is known for Outer Web Developers as Mojolicious team?

### DISCUSSION THREAD

**Discussion by yuki-kimoto** [mojolicious/mojo#1731] [General]
What messages is known for Outer Web Developers as Mojolicious team?

**Comment by jberger** [comment]
I don't understand the question. Can you please rephrase?

**Reply by yuki-kimoto** [reply]
I think this discussion is a bulletin board for those who are <b>already</b> Mojolicious users.

Do Mojolicious team have messages to talk to users who <b>don't know Mojolicious</b> yet and want to do web development?
